examples/pytorch/decision_transformer/run_decision_transformer.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
```

Comment:
We don't usually put examples that work for just one model in the examples folder, but in a research project. I think this one should be moved there.